src/app/beer_garden/__init__.py

```diff
@@ -36,6 +44,33 @@ def signal_handler(signal_number, stack_frame):
     beer_garden.logger.info("OK, we're all shut down. Have a good night!")
 
 
+# def establish_events_manager():
+#     global events_manager
+#     events_manager = EventsManager()
+#
+#     global events_queue
+#     events_queue = Queue()
+#
+#     events_manager.set_queue(events_queue)
+#
+#     event_config = beer_garden.config.get("event")
+#     if event_config.parent.http.enable:
+#         beer_garden.events_manager.register_listener(
+#             ParentProcessor(event_config.parent.http)
+#         )
+#
+#     beer_garden.events_manager.start()
+
+def establish_events_queue(queue: Queue = None):
+    global events_queue
+
+    if queue is None:
+        context = multiprocessing.get_context("spawn")
+        queue = context.Queue()
+
+    events_queue = queue
+
+
```

Comment:
I know we use this singleton pattern for other things, but that's mainly to avoid getting into a dependency-injection nightmare.

To me, having both the 'main' app and the entry points calling the same method make it less clear what's going on.

Reply:
Both Entry Points and the App need to call the same function. That way everything is configured the same for the Events Handler. I pulled it out of the Main App, but let's chat about the pro's and con's about them calling the same method.